dist/demo-app/README.md

```diff
@@ -1,74 +1,27 @@
 # Capital Framework
 
-A Capital Framework demo project. Use grunt to process the source files in `src/` and output them to `dist/`:
+A Capital Framework sample project.
+To preview it, run `python -m SimpleHTTPServer` and open http://localhost:8000 in a web browser.
```

Comment:
Mmm, good point... Is `index.html` file the same file as `src/index.html` that comes out of the generator? Or is it customized for this downloadable archive?